test/acceptance/exception_rescue_test.rb

```diff
@@ -48,7 +48,7 @@ def test_unsatisfied_expectation_exception_is_not_caught_by_standard_rescue
     assert_equal [
       'not all expectations were satisfied',
       'unsatisfied expectations:',
-      '- expected exactly once, not yet invoked: #<Mock:mock>.some_method(any_parameters)'
+      '- expected exactly once, invoked never: #<Mock:mock>.some_method(any_parameters)'
```

Comment:
Minor: I think "expected exactly once, invoked never" reads slightly awkwardly compared to "expected exactly once, not yet invoked" or "expected exactly once, never invoked".  However, given that it simplifies the logic, it's probably good enough.